torch/_inductor/lowering.py

```diff
@@ -1491,7 +1491,7 @@ def apply_constraint(arg, fx_arg):
 make_fallback(aten.topk)
 make_fallback(aten.upsample_bicubic2d_backward, require_contiguous)
 
-make_fallback(aten.view_as_complex.default, require_contiguous)
+make_fallback(aten.view_as_complex, require_contiguous)
```

Comment:
We shouldn't need this change to make it work, they are equivalent

Reply:
Without that change we get

```
LoweringException: AssertionError: view_as_complex.default is not supported with cpp wrapper
```

Reply:
It's being fixed in https://github.com/pytorch/pytorch/issues/102752. Let's not duplicate work.